docker/Dockerfile

```
@@ -0,0 +1,25 @@
+# Build the source code
+FROM node:14-alpine AS builder
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm ci
+COPY tsconfig*.json ./
+COPY src src
+RUN npm run build
+
+FROM buildkite/puppeteer
+
+WORKDIR /opt/nvidia-snatcher/
+
+# Copy dependecies definition and configuration files
+COPY package.json package-lock.json ./
+COPY tsconfig*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy compiled files
+COPY --from=builder /usr/src/app/build/ .
+
+# Set the start script as entrypoint
+CMD node index.js
```

No need to copy over `package.json` or `tsconfig.json`.

```suggestion
COPY --from=builder /build/node_modules/ node_modules/
COPY --from=builder /build/build/ build/

CMD [ "node", "./build/index.js" ]
```